cmd/hierarchy-transformer/output/businessInvestment/hierarchy.cypher

```diff
@@ -0,0 +1,175 @@
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "G" and child.code = "G45" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "G" and child.code = "G46" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "G" and child.code = "G47" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "_T" and child.code = "H" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "H" and child.code = "H49" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "H" and child.code = "H50" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "H" and child.code = "H51" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "H" and child.code = "H52" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "H" and child.code = "H53" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "_T" and child.code = "I" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "_T" and child.code = "J" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "J" and child.code = "J58" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "J" and child.code = "J58T60" CREATE (child)-[:hasParent]->(parent);
```

Comment:
Should this be the one that links directly to J, and J58 and J59_60 go through this?